### DISCUSSION TITLE
Can not get it to work

### DISCUSSION THREAD

**Discussion by josegomes123** [umstek/tailwindcss-glow#8] [Q&A]
I've tried configuring this plugin, the provided config doesn't seem to be valid, I got it to work after fixing the parentheses error and I tried using the plugin with the default tags but nothing appears to be working. I am kinda new to tailwind so it might be my fault

Can you help me out? Maybe update the readme file with some usage examples?

**Comment by umstek** [comment]
Hi,
This plugin is for tailwind 1.x and probably won't work with 2.x. I haven't started the work for tailwind 2.x. Planning to do this soon.
